avocado/utils/build.py

```diff
@@ -18,6 +18,36 @@
 from . import process
 
 
+def configure(path, configure=None):
```

Comment:
With this we are nicely hitting the limitation of non-object-oriented `build` library. How nice it'd be to say:

```python
with build.Build("/path/to/kernel/src") as kernel:
    kernel.configure("--with something")
    kernel.make("-j 10")
    kernel.make("install")
```

Anyway I see the benefit in using this in case we're using mutliple versions within one test, one with "./configure", the other with automake. Perhaps we should add cmake as well, what do you think?

Reply:
Also the history taught us people might be interested in the output etc. How about creating `run_configure` and `configure`?

Reply:
I'd defer having `run_configure` until needed.